test/index.test.js

```diff
@@ -1,27 +1,203 @@
 import test from 'ava';
+import {oneLineTrim} from 'common-tags';
```

Comment:
we should use `oneLine` because `oneLineTrim` removes spaces between words

Reply:
Good catch!